marimbabot_audio/package.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marimbabot_audio</name>
+  <version>0.0.0</version>
+  <description>The marimbabot_audio package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="root@todo.todo">root</maintainer>
```

Comment:
Maybe the email of somebody that wanted to do audio

Reply:
depending on whether the person wants to have their email publically avaiblabe yes we could do that

Reply:
depending on whether the person wants to have their email publically avaiblabe

Reply:
The mail is exposed either way due to git

Reply:
afaik it is not. You can set "Keep my email address private" inside the e-mail settings. We should figure this out within the group

Reply:
This setting adds a dummy mail to the commit in your case `Juphex <20795522+Juphex@users.noreply.github.com>`. Be we can discuss this later on.